source/reference/api/start.txt

```diff
@@ -140,7 +140,12 @@ Request Body Parameters
        accepts writes.
 
        To reverse sync, the ``enableUserWriteBlocking`` field must be set
-       to ``true``.
+       to ``true``. To allow the source cluster to accept writes again, run
```

Comment:
Hey @kanchana-mongodb, does it make sense to include the context @ivangrigolon mentions in the comments?